EDDiscovery/UserControls/UserControlScanGrid.Designer.cs

```diff
@@ -0,0 +1,178 @@
+            this.vScrollBarCustom2.ThumbButtonColor = System.Drawing.Color.DarkBlue;
+            this.vScrollBarCustom2.ThumbColorScaling = 0.5F;
+            this.vScrollBarCustom2.ThumbDrawAngle = 0F;
+            this.vScrollBarCustom2.Value = -1;
+            this.vScrollBarCustom2.ValueLimited = -1;
+            // 
+            // img
+            // 
+            dataGridViewCellStyle1.Alignment = System.Windows.Forms.DataGridViewContentAlignment.MiddleCenter;
+            dataGridViewCellStyle1.NullValue = ((object)(resources.GetObject("dataGridViewCellStyle1.NullValue")));
+            dataGridViewCellStyle1.WrapMode = System.Windows.Forms.DataGridViewTriState.False;
+            this.img.DefaultCellStyle = dataGridViewCellStyle1;
+            this.img.FillWeight = 5F;
+            this.img.HeaderText = "";
+            this.img.Name = "img";
+            this.img.Resizable = System.Windows.Forms.DataGridViewTriState.True;
+            this.img.SortMode = System.Windows.Forms.DataGridViewColumnSortMode.Automatic;
+            // 
+            // BodyName
+            // 
+            this.BodyName.FillWeight = 15F;
+            this.BodyName.HeaderText = "Name";
+            this.BodyName.MinimumWidth = 20;
+            this.BodyName.Name = "BodyName";
+            // 
+            // BodyClass
+            // 
+            this.BodyClass.FillWeight = 25F;
+            this.BodyClass.HeaderText = "Class";
+            this.BodyClass.MinimumWidth = 20;
+            this.BodyClass.Name = "BodyClass";
+            // 
+            // BodyDetails
+            // 
+            this.BodyDetails.FillWeight = 50F;
+            this.BodyDetails.HeaderText = "Briefing";
+            this.BodyDetails.MinimumWidth = 30;
+            this.BodyDetails.Name = "BodyDetails";
+            // 
+            // UserControlScanGrid
+            // 
+            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.Controls.Add(this.dataViewScrollerPanel2);
+            this.Name = "UserControlScanGrid";
+            this.Size = new System.Drawing.Size(572, 572);
+            this.dataViewScrollerPanel2.ResumeLayout(false);
+            ((System.ComponentModel.ISupportInitialize)(this.dataGridViewScangrid)).EndInit();
+            this.ResumeLayout(false);
+
+        }
+
+        #endregion
+        private ExtendedControls.DataViewScrollerPanel dataViewScrollerPanel2;
+        private ExtendedControls.VScrollBarCustom vScrollBarCustom2;
+        private System.Windows.Forms.DataGridView dataGridViewScangrid;
+        private System.Windows.Forms.DataGridViewImageColumn img;
+        private System.Windows.Forms.DataGridViewTextBoxColumn BodyName;
+        private System.Windows.Forms.DataGridViewTextBoxColumn BodyClass;
+        private System.Windows.Forms.DataGridViewTextBoxColumn BodyDetails;
```

Comment:
I've found that it's a good idea to give Controls a name that represents their type or intended purpose. `img` sounds like it should be an `Image`, and is something that one would use for a temporary variable name when resizing an image or something. Try renaming the columns to something like `{colImage, colName, colClass, colDetails}` or `{columnImage, columnName, etc.}`. It makes it a little clearer to outside observers what they are looking at if they aren't staring at bottom of the .Designer.cs file.

Reply:
Thanks!

Reply:
Done.